getting-started/templates/systemlink-values.yaml

```diff
@@ -902,3 +902,22 @@ comments:
   ## Maximum number of comments supported per delete comments request.
   ##
   maximumCommentsPerDeleteRequest: 1000
+
+## Configuration for specification management service.
+##
+specificationmanagement:
```

Comment:
If we do not expect a typical customer to need to change these settings, do not put them in this file. This file should be kept to just key decisions that a customer needs to make when deploying the product. Adding unnecessary detail just makes it unapproachable.

Reply:
Sure, typically customers may not want to change those configurations. It can be included if required in future

Reply:
I have removed it